azext_iot/central/_help.py

```diff
@@ -316,6 +316,20 @@ def _load_central_monitors_help():
             az iot central app validate-messages --app-id {app_id} -d {device_id} --cg {consumer_group_name}
     """
 
+    helps[
+        "iot central app monitor-properties"
+    ] = """
+        type: command
+        short-summary: Monitor desired and reported  sent to/from the IoT Hub for an IoT Central app.
+        long-summary: |
```

Comment:
For long description, it would be helpful for a brief/high-level summary of how the property monitoring works - users know what to expect from reading the description.